app/wama.web/Views/AdminConsole/UserTool/Index.cshtml

```diff
@@ -3,8 +3,37 @@
 
 <h2>User account summaries</h2>
 
-<h3>Accounts pending approval</h3>
-@Html.Partial("UserAccountTableDisplay")
 
-<h3>Suspended Accounts</h3>
-@Html.Partial("UserAccountTableDisplay")
+
+<center>
```

Comment:
use bootstrap css class for centering the h3 tag ('text-center'). Remove center tag